_posts/2016/2016-12-05-election-sue-mcclatchy.md

```diff
@@ -0,0 +1,79 @@
+This year I secured 4 years of partnership funding between 
+[The Jackson Laboratory] (http://www.jax.org) and the Carpentries. The first year of
+partnership is funded by an internal grant, with succeeding years funded by a grant from 
+the U.S. National Institutes of Health (NIH).
+
+How I can contribute
+-----
+
+I bring an uncommon perspective from formal teacher training and an 8-year career as a
+K-12 teacher in the U.S. and Latin America. Diverse abilities and cultures in the 
+classroom are the norm for me, and I have much to share with instructors about how to best 
+meet everyone's needs - the first thing being a focus on learners' needs. My training 
+instilled the idea that the learner, not the instructor, is the most important person in 
+the room. A learner-centered approach to instruction responds well to diversity and arms 
+the instructor with tools to adapt instruction to new situations and new people. Many of 
+these tools work equally well with both child and adult learners, and work across cultures 
+and abilities. 
+
+I entered biomedical research in the early 2000s and have contributed my teaching expertise
+to the field since then. I'm well-acquainted with the need for improved computing and data
+analysis skills in research and know that the Carpentry approach promotes greater research
+productivity and happiness. I'll bring this understanding to instructor training and 
+mentoring to bolster instructional expertise within the Carpentry community.
+
+I will help Software Carpentry to expand its training and instructional footprint into 
+new regions, especially in Latin America, by teaching and by mentoring instructors there. 
+In January 2017 I will teach  workshops at the Talleres Internacionales de Bioinformática 
+(International Bioinformatics Workshops) in Cuernavaca, Morelos, México. I intend to 
+follow this with further training in Latin American countries, and to support those
+already teaching in these countries.
+
+I will contribute grant-writing expertise to grow and sustain Software Carpentry by 
+identifying and pursuing new sources of funding from foundations, government grants and 
+institutional partnerships. I'm presently working on a NIH grant proposal and have 
+shared the proposal with key NIH staff. I'm especially interested in pursuing funding
+that will broaden Software Carpentry into different communities than those it already 
+represents well.
+
+My goals are the following:
+-----
+
+1. **Bolster instructional capacity and expertise** by training and mentoring new 
+instructors in all disciplines.
+2. **Broaden Software Carpentry's reach** into largely untapped regions, especially in 
+Latin America.
+3. **Build Software Carpentry** into a sustainable, well-funded organization that reaches
+a diverse audience.
+
+More about me if you're so inclined
+-----
+More about me on [LinkedIn] (https://www.linkedin.com/in/suemcclatchy), 
```

Comment:
The whitespace between ] and ( is not allow in Markdown.  I fixed it at 59fac96. cc @smcclatchy